hack/manifest-gen/manifest-gen.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+# Script to generate an ECK installation manifest
+
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+CHART_DIR="${SCRIPT_DIR}/assets/charts/eck"
+
+update_chart() {
+    local ALL_CRDS="${SCRIPT_DIR}/../../config/crds/all-crds.yaml"
+    local VERSION
+    VERSION=$(cat "${SCRIPT_DIR}/../../VERSION")
+
+    cp -f "$ALL_CRDS" "${CHART_DIR}/crds/all-crds.yaml"
+    sed -i -E "s#version: [0-9]+\.[0-9]+\.[0-9]+#version: $VERSION#" "${CHART_DIR}/values.yaml"
+    sed -i -E "s#appVersion: [0-9]+\.[0-9]+\.[0-9]+#appVersion: $VERSION#" "${CHART_DIR}/Chart.yaml"
+}
+
+usage() {
+    echo "Usage: $0 [-u | -g <args>]"
+    echo "    '-u'"
+    echo "         Update the chart and exit"
```

Comment:
it's immediately clear to me what "update the chart" means, can we clarify a bit?